src/varibales.css

```diff
@@ -0,0 +1,5 @@
+:root {
```

Comment:
there are two variable files?

Reply:
Hi, Edrees. thank you for noticing, one of them had a typing mistake on the file name. I fixed it but made a mistake while committing. This is why old one stayed also. Fixed it now.